lib/ppx/ppx_js.cppo.ml

```diff
@@ -0,0 +1,412 @@
+open Ast_mapper
+open Ast_helper
+open Asttypes
+open Parsetree
+
+open Ast_convenience
+
+(** Check if an expression is an identifier and returns it.
+    Raise a Location.error if it's not.
+*)
+let exp_to_string = function
+  | {pexp_desc= Pexp_ident {txt = Longident.Lident s}} -> s
```

Comment:
`let f x = x##A = 0`
has type
`val f : < A : int Js.meth; .. > Js.t -> 'jsoo_bcb1e9dd`.
But this type cannot be reread by ocaml because method name cannot start with capital letter